src/main/java/org/smartregister/converters/PhysicalLocationAndStocksConverter.java

```diff
@@ -0,0 +1,32 @@
+package org.smartregister.converters;
+
+import com.ibm.fhir.model.resource.Bundle;
+import com.ibm.fhir.model.resource.Location;
+import com.ibm.fhir.model.type.code.BundleType;
+import org.smartregister.domain.PhysicalLocationAndStocks;
+import org.smartregister.domain.Stock;
+import org.smartregister.domain.StockAndProductDetails;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PhysicalLocationAndStocksConverter {
+
+	public static Bundle convertLocationAndStocksToBundleResource(PhysicalLocationAndStocks physicalLocationAndStocks) {
+		Location location = LocationConverter.convertPhysicalLocationToLocationResource(physicalLocationAndStocks);
+		Bundle.Builder bundleBuilder = Bundle.builder();
```

Comment:
Probably used `DomainResource.contained` instead of a Bundle. Bundle is like a collection or list. This would also match the new model `PhysicalLocationAndStocks`

Reply:
I checked this out and the definition of contained resource states `These resources do not have an independent existence apart from the resource that contains them - they cannot be identified independently, and nor can they have their own independent transaction scope.` So since we can independently identify a stock resource i thought Bundle is the better option  i had.

Reply:
Cool